README.md

```diff
@@ -1,11 +1,14 @@
 # Melange
 
-A _mixture_ of tooling combined to produce JS from OCaml / Reason.
+**Tools for producing JS from OCaml / Reason.**
```

Comment:
how about:

> A toolchain that compiles OCaml / Reason to JavaScript

Reply:
Well, it's not an entire toolchain. Maybe just 'tooling'? Or more specific as 'Compiler'? 'Mixture of tooling' was a bit nondescriptive. It might make sense for devs but not for simple users.

Reply:
Mixture of tooling was a pun on [melange](https://www.google.com/search?q=define%3Amelange&oq=define%3Amelange&aqs=chrome..69i57j69i58.3321j0j1&sourceid=chrome&ie=UTF-8).

Reply:
I didn't catch that. I try to use that somewhere further down.